run_automation.py

```diff
@@ -12,10 +12,10 @@
         sleep(3)
         continue
 
-    package = device.get_top_activity_package()
-    if package != app_package:
-        print('App is not running')
-        continue
+    #package = device.get_top_activity_package()
+    #if package != app_package:
+    #    print('App is not running')
+    #    continue
```

Comment:
I also noticed that this doesn't work as expected anymore...
My fix was to change the function get_top_activity_package() in android_device.py like this:
`    def get_top_activity_package(self):
        return next(filter(
            lambda x: x.package == "com.TechTreeGames.TheTower",
            self.device.get_top_activities())).package
`
I think it is pretty straightforward what this does.